src/vmm/src/devices/virtio/queue.rs

```diff
@@ -308,6 +308,15 @@ impl Queue {
         Ok(slice.ptr_guard_mut().as_ptr())
     }
 
+    fn log_queue_component_address(component: &'static str, addr: GuestAddress, size: usize) {
+        log::info!(
+            "Placed virt queue {} at [{}, {})",
```

Comment:
nit: I thought we call the combination of desc table and avail/used rings a virt queue. A desc table on its own doesn't look like a queue to me.

Reply:
ok, got it now, thx.